NetKAN/HullcamVDS.netkan

```diff
@@ -10,9 +10,19 @@
     "install" : [
         {
             "file"       : "HullCameraVDS",
-            "install_to" : "GameData"
+            "install_to" : "GameData",
+            "filter": [
+                "Thumbs.db",
+                "Thumbs.db:encryptable"
+            ],
+            "filter_regexp": [
```

Comment:
A filter applies to a `file` or `find`. This particular filter has nothing to affect because it's sitting on its own. Additionally we don't need to filter anything outside of the scope of our `file` or `find`. Good spot on the `Thumbs.db` files though. :+1: